scripts/_common.sh

```diff
@@ -3,7 +3,10 @@
 # COMMON VARIABLES
 #=================================================
 
-pkg_dependencies="php-gd php-json php-intl php-mcrypt php-curl php-apcu php-redis php-ldap php-imagick php-zip php-mbstring php-xml imagemagick acl tar smbclient at"
+pkg_dependencies="imagemagick acl tar smbclient at"
+
+php_version="7.2"
```

Comment:
Maybe to use `$YNH_PHP_VERSION` instead, https://github.com/YunoHost/yunohost/pull/880/files#diff-9d1bacd64942317e00cbf97e9142934cR5

Reply:
$YNH_PHP_VERSION is not in the current version.

Reply:
It doesn't have to.
It just that, if the variable is used now, we won't have to change it later.

Reply:
OK, so we override YNH_PHP_VERSION in the _common.sh (and update the PHP helpers), preventing use from specifying the PHP version to use for the PHP helpers.

What about the php CLI calls? Do we keep php${php_version} or do we adapt to php${YNH_PHP_VERSION}?

Reply:
For the current PR, changing `php_version` by `YNH_PHP_VERSION` will only be a modification of the name of the variable.
But later, the official helper, that use as well `YNH_PHP_VERSION` won't override the variable since it's already declared into the _common.sh
As said here (https://github.com/YunoHost/example_ynh/pull/110#issuecomment-583871721), 
> "A packager willing to use a custom version of php will set `YNH_PHP_VERSION=7.4` into his `_common.sh` and use `$YNH_PHP_VERSION` instead of `$YNH_DEFAULT_PHP_VERSION` into the scripts."

Reply:
OK, so I assume we adapt to php${YNH_PHP_VERSION} for CLI calls.

Reply:
Indeed, that was also was I intended to say ;)